docs/schemas/Marmoset-Metadata/HMBA Marmoset Population.csv

```diff
@@ -0,0 +1,45 @@
+﻿Proposed BICAN Field Name,BICAN UUID,Label,Variable (Short) Name,Definition,Short Description,Human Readable Label,is_required,Permissible Values (incl Null),Description (Permissible Values),Data Type,Version Number,Version Date,Sub Domain Name,Domain Name,Size,Input Restrictions,Min Value,Max Value,Measurement Type
+Local Donor ID,,Local Donor ID,subject_id,,,,,,,text,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Sex at Birth,,Sex at Birth,sex,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Age Value (Years),,Age Value (Years),age_of_death,,,,,,,numeric,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Date of Death,,Date of Death,year_of_death,,,,,,,date,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Manner of Death,,Manner of Death,manner_of_death,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Mating Status,,Mating Status,mating_status,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Medical Records Available,,Medical Records Available,medical_records_available,,,,,,,exclusive_categorical,,,General Subject Fields,HMBA_Marmoset_Population,,,,,
+Hemisphere,,Hemisphere,hemisphere,,,,,,,exclusive_categorical,,,General Specimen Data,HMBA_Marmoset_Population,,,,,

```

Comment:
do they have only one hemisphere that it has to be specified?

Reply:
It can be both, I suppose, but that is managed in the controlled values and NIMP.